privacyidea/api/lib/prepolicy.py

```diff
@@ -1222,8 +1222,7 @@ def is_remote_user_allowed(req):
                                                          scope=SCOPE.WEBUI,
                                                          user=loginname,
                                                          realm=realm,
-                                                         client=g.client_ip,
-                                                         audit_data=g.audit_object.audit_data)
+                                                         client=g.client_ip)
```

Comment:
``is_remote_user_allowed`` is not only called by ``single_page_application`` but also by ``get_auth_token``, which implements the ``/auth`` endpoint. In the latter case, I think we do have an audit object and we should pass it to ``get_action_values`` to extend the audit log entry with the information that the REMOTE_USER policy is in effect.

Reply:
Done in 0cbab21b